libs/components/forms/src/lib/modules/file-attachment/file-drop/file-drop.component.html

```diff
@@ -169,11 +169,14 @@
     </div>
   </div>
 </fieldset>
+<!-- TODO: once we add control value accessor, populate the below `sky-form-errors` with the appropriate values for `showErrors`, `touched`, and `dirty`-->
+<!-- for now we are marking touched as true so that file drop will continue to show `required` error-->
```

Comment:
ill confirm with kerry if such a story exists, but either way. i prefer to keep this here so that if/when we do get to that story this is addressed as whoever takes that story will see the todo at that point

Reply:
Story: https://dev.azure.com/blackbaud/Products/_workitems/edit/3053845 😎 

Another reason we shouldn't leave TODO's in the code: comments are not removed and are published with the bundle, needlessly increasing its size. 
<img width="591" alt="Screenshot 2024-08-22 at 3 17 54 PM" src="https://github.com/user-attachments/assets/8ad92a45-8f09-476a-9f93-c82420572bac">

If you absolutely must keep the comment, we should at the very least reword it to not include `showErrors`. 👍